app/models/webauthn_verification.rb

```diff
@@ -0,0 +1,7 @@
+class WebauthnVerification < ApplicationRecord
+  belongs_to :user
+
+  validates :user_id, uniqueness: true
```

Comment:
@jchestershopify I removed the presence validation since `belongs_to` requires user [by default](https://edgeguides.rubyonrails.org/configuring.html#config-active-record-belongs-to-required-by-default).

Reply:
Nice catch!